python/tests/prototype/chain.py

```diff
@@ -16,29 +16,23 @@
 from gtsam import Pose3
 
 
-def compose(A: Tuple[Pose3, np.ndarray], B: Tuple[Pose3, np.ndarray]):
-    """Monoid operation for pose,Jacobian pairs."""
-    aTb, Jb = A
-    bTc, Jc = B
-    assert Jb.shape[0] == 6 and Jc.shape[0] == 6, f"{Jb.shape} and {Jc.shape}"
+def compose(aSbj: Tuple[Pose3, np.ndarray], bSck: Tuple[Pose3, np.ndarray]):
+    """Monoid operation for chains, i.e., pose,Jacobian pairs."""
+    aTb, bAj = aSbj
+    bTc, cAk = bSck
+    assert bAj.shape[0] == 6 and cAk.shape[0] == 6,\
+        f"Jaocobains should have 6 rows, sapes are {bAj.shape} and {cAk.shape}"
```

Comment:
Typos: Jaocobains -> Jacobians and sapes -> shapes

Reply:
Thanks! I'll take this as "approve" and merge. Note there is a button you should use to "approve" a PR :-)

Reply:
Oops, sorry about that, now I know :-)